plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMigrateProcedure.java

```diff
@@ -45,9 +48,10 @@ protected QueryRunner createQueryRunner()
         DistributedQueryRunner queryRunner = IcebergQueryRunner.builder().setMetastoreDirectory(dataDirectory.toFile()).build();
         queryRunner.installPlugin(new TestingHivePlugin());
         queryRunner.createCatalog("hive", "hive", ImmutableMap.<String, String>builder()
-                        .put("hive.metastore", "file")
-                        .put("hive.metastore.catalog.dir", dataDirectory.toString())
-                        .put("hive.security", "allow-all")
+                .put("hive.metastore", "file")
+                .put("hive.metastore.catalog.dir", dataDirectory.toString())
+                .put("hive.security", "allow-all")
+                .put("hive.parquet.time-zone", DateTimeZone.forID("America/Bahia_Banderas").getID())
```

Comment:
This config property doesn't change result as far as I tried.

Reply:
I need this setting to get const offset between writing local date in Hive and reading data in UTC in Iceberg. (Parquet only)

Reply:
> The setting probably is not needed when you add the JVM argument -Duser.timezone=America/Bahia_Banderas

It's defined in root pom.xml. I would recommend removing `hive.parquet.time-zone` from this test class.
https://github.com/trinodb/trino/blob/6347248a255305ce0a4924ede4390aecfd62115f/pom.xml#L88